### ISSUE TITLE
Request: Support for KP125 Energy Monitor Smart Plug

### ISSUE THREAD

**Issue by PlatGitHub468** [plasticrake/homebridge-tplink-smarthome#226]
I just bought two Kasa KP125 Smart Plug with Energy Monitor from BestBuy.  The current HomeBridge plugin does not list this as a support model.  Would you please add support to KP125 so that we can see Energy feature in apps such as Eve App?

### Versions

- **Node**:       `node --version`
- **Homebridge**: `homebridge --version`
- **OS**:         

### Configuration

```json

```

### Homebridge Log / Command Output

```

```

**Comment by plasticrake** [comment]
Did you try it? Most new devices work with this plugin

**Comment by PlatGitHub468** [comment]
Yes, I tried this plugin with HomeBridge V4.41.5 and Homebridge-tplink-smarthome v6.4.0.  However, when using the Eve App, the new KP125 Plug does not have the energy feature like the Eve SmartPlug for example,  My request that in your plugin, somehow enable the energy feature so in the Eve App, I can see how much power is consumed .

**Comment by lfbettcher** [comment]
The equivalent EP25 plug also does not show energy in Eve.
